.changeset/nine-garlics-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/api': patch
+---
+
+Fixed the check for remaining admins to allow updates of associated users via role update
```

Comment:
Mhh, how about:

```suggestion
Resolved error indicating inability to remove last admin user during updates
```

Reply:
I like it! <3 Can we add something like "... during user updates via roles" at the end?